core/src/test/java/io/undertow/server/security/SimpleConfidentialRedirectTestCase.java

```diff
@@ -85,7 +85,7 @@ public void simpleRedirectTestCase() throws IOException, GeneralSecurityExceptio
             sendRequest(client, "/foo+bar", null);
             sendRequest(client, "/foo+bar;aa", null);
             sendRequest(client, "/foo+bar;aa", "x=y");
-            sendRequest(client, "/foo+bar%3Aaa", "x=%3Ablah");
+            sendRequest(client, "/foo+bar:aa", "x=%3Ablah");
```

Comment:
Why this change? Isn't this supposed to test https://issues.redhat.com/browse/UNDERTOW-1565?

Reply:
@jstourac  this is a result of the upgrade to Jakarta spec. I don't think it is a problem, because "/foo+bar:aa" is the output, and to be that's consistent to the remaining of the other expected results, but I might be mistaken.

Reply:
@jstourac  I figured what is causing this, I'll submit a fix pretty soon

Reply:
and, btw, you're right, thankfully the problem is in the test and not in Undertow

Reply:
@jstourac  done! The test now passes again